arrow-libs/core/arrow-core/src/test/kotlin/arrow/core/EitherTest.kt

```diff
@@ -323,3 +454,12 @@ private suspend fun <A> throwException(
   a: A
 ): Either<Throwable, Any> =
   throw RuntimeException("An Exception is thrown while handling the result of the supplied function.")
+
+private fun forAllSmallInt(fn: PropertyContext.(a: Int) -> Boolean) =
```

Comment:
I believe we have some other utilities like these somewhere. Perhaps these functions should be moved there and made public or internal.